### PR TITLE
py sol for creating strings II and some typo

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by freakin23** [comment]
AC: 
Screenshot from 2024-09-07 13-23-29

File: solutions/gold/cses-1715.mdx [hunk 1]
```diff
@@ -66,4 +70,56 @@ int main() {
...
+	inv[MAXN] = exp(fac[MAXN], MOD - 2, MOD)
+	for i in range(MAXN, 0, -1):
+		inv[i - 1] = inv[i] * i % MOD
+
+
+# EndCodeSnip
+
+st = input()
+cnt = Counter(st)
+factorial()
```
Comment (TheGamingMousse) [inline solutions/gold/cses-1715.mdx:113]: did you really need to make a separate method for those lol

Comment (freakin23) [reply solutions/gold/cses-1715.mdx:113]: The template was already used in several solutions so ig why not use it rather than something new

Comment (TheGamingMousse) [reply solutions/gold/cses-1715.mdx:113]: oh ic it is in the module, fair enough

File: solutions/gold/cses-1715.mdx [hunk 2]
```diff
@@ -66,4 +70,56 @@ int main() {
...
+def inverses():
+	inv[MAXN] = exp(fac[MAXN], MOD - 2, MOD)
+	for i in range(MAXN, 0, -1):
+		inv[i - 1] = inv[i] * i % MOD
+
+
+# EndCodeSnip
+
+st = input()
+cnt = Counter(st)
```
Comment (SansPapyrus683) [inline solutions/gold/cses-1715.mdx:112]: make variable names consistent

Comment (freakin23) [reply solutions/gold/cses-1715.mdx]: done

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by SansPapyrus683** [comment]
show ac ss again

**Comment by freakin23** [comment]
> show ac ss again

Screenshot from 2024-09-09 22-11-04

**Pull request merged**